x-pack/plugins/observability/public/components/shared/exploratory_view/header/add_to_case_action.test.tsx

```diff
@@ -21,6 +24,31 @@ describe('AddToCaseAction', function () {
     expect(await findByText('Add to case')).toBeInTheDocument();
   });
 
+  it('should parse relative data to the useAddToCase hook', async function () {
+    const useAddToCaseHook = jest.spyOn(useCaseHook, 'useAddToCase');
+    jest.spyOn(datePicker, 'parseRelativeDate').mockReturnValue(moment('2021-11-10T10:52:06.091Z'));
```

Comment:
Thanks for adding this test! It LGTM.

As a nit in case you want a tip: I think here we're kinda losing some value by mocking early on `parseRelativeDate`. Instead we could mock the global timers setting a fixed date and then the parsed dates in `timeRange` would always be the same, but they would indeed be `now` and `5 minutes ago`.

Currently, both dates are the same as we always return the same mock.

I won't block because of this though, just a tip in case you want the test to be closer to runtime behaviour.

In case you wanna read more about timers I'd recommend [this link](https://jestjs.io/docs/timer-mocks). Alternatively you can use [`jest-when`](https://jestjs.io/docs/timer-mocks) for returning different values for the passed dates, but then I think we end-up in the same place as before as it's still a tightly coupled mock.